terraform/000-core/main.tf

```diff
@@ -16,9 +15,8 @@ module "ecscluster" {
 module "ecscluster_secondary" {
   count = var.aws_region_secondary == "" ? 0 : 1
 
-  source   = "github.com/silinternational/terraform-modules//aws/ecs/cluster?ref=8.0.1"
-  app_name = var.app_name
-  app_env  = "secondary"
+  source       = "github.com/silinternational/terraform-modules//aws/ecs/cluster?ref=8.1.0"
+  cluster_name = "idp-secondary"
```

Comment:
Do we really want this hard coded, one-size-fits-all?
Maybe "${var.app_name}-${var.app_env}-secondary" ?

Reply:
Or just "${var.app_name}-secondary"

Reply:
Yeah, I don't know what I was thinking!